### ISSUE TITLE
Port to Python 3

### ISSUE THREAD

**Issue by cchristelis** [kartoza/projecta#875]
# Problem

We need to handle unicode characters in various places, porting to python 3 would solve these issues.

**Comment by timlinux** [comment]
Agreed!

**Comment by ismailsunni** [comment]
Agree, but we need a plan for this, and target when do we want to do it. 20 months before python 2 support ended.
